### PR TITLE
Fix incorrect specification of SystemAgent

### PR DESCRIPTION
**Pull request by tommy5oo**

### CODE REVIEW

File: nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultSystemAgent.kt [hunk 1]
```diff
@@ -338,16 +338,9 @@ class DefaultSystemAgent(
         Logger.d(TAG, "[handleResetConnection] $info")
         val payload =
             MessageFactory.create(info.directive.payload, ResetConnectionPayload::class.java)
```
Comment (shshshl09) [inline nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultSystemAgent.kt:340]: maybe...description field is optional?

https://tde.sktelecom.com/wiki/display/ERECTUS/2.+System+Interface+1.3#id-2.SystemInterface1.3-7)ResetConnectiondirective

Comment (tommy5oo) [reply nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultSystemAgent.kt:340]: yes. it's optional.

Comment (tommy5oo) [reply nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultSystemAgent.kt:340]: Oh.. I'll change the description to nullable..

Comment (shshshl09) [reply nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultSystemAgent.kt:340]: maybe need payload nullable? following:

if (payload == null) {

Comment (tommy5oo) [reply nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultSystemAgent.kt:340]: val payload = MessageFactory.create("{}", ResetConnectionPayload::class.java)
AssertTrue(payload == null)

Comment (tommy5oo) [reply nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultSystemAgent.kt]: Fixed. thanks.

**Review by shshshl09 — APPROVED** [review]

**Pull request merged**